File: P120864.py
def solution(my_string):
    answer = 0
    tmp = "0"
    for digit in my_string:
        if ord("A") <= ord(digit) <= ord("z"):
            answer += int(tmp)
            tmp = "0"
        else:
            tmp += digit
    answer += int(tmp)
    return answer

File: P120864.java
class P120864 {
    public int solution(String my_string) {
        int answer = 0;
        String tmp = "0";
        for (int i=0; i<my_string.length(); i++) {
            char digit = my_string.charAt(i);
            if ('A' <= digit && digit <= 'z') {
                answer += Integer.parseInt(tmp);
                tmp = "0";
            }
            else tmp += digit;
        }
        answer += Integer.parseInt(tmp);
        return answer;
    }
}